databricks/koalas/frame.py

```diff
@@ -5158,10 +5151,6 @@ def replace(
         value : int, float, string, or list
```

Comment:
Can we update the doc of `to_replace` argument above to say `dict` is supported?

Reply:
I've updated it to include `dict` and have added usage to the description below.